.eslintrc.json

```diff
@@ -15,11 +15,14 @@
       "no-console": 1,
       "indent": [ 1, 2 ],
       "comma-dangle": 0,
+      "max-len": "off",
```

Comment:
@urbanishimwe  try to de- activate this in a single file so that it doesn't affect all our files